.rubocop_todo.yml

```diff
@@ -4,10 +4,15 @@ Security/MarshalLoad:
   Exclude:
     - 'app/models/concerns/hyrax/user.rb'
 
+Lint/HandleExceptions:
+  Exclude:
+    - 'app/models/concerns/hyrax/admin_set_behavior.rb'
```

Comment:
This is not required since you are wrapping the method with `rubocop:disable ...`